motoko/threshold-schnorr/README.md

```diff
@@ -33,23 +25,22 @@ Motoko programming language. There is also a
 [Rust](https://github.com/dfinity/examples/tree/master/rust/threshold-schnorr)
 version available in the same repo and follows the same commands for deploying.
 
-
-## Prerequisites
--   [x] Download and [install the IC
-    SDK](https://internetcomputer.org/docs/current/developer-docs/setup/index.md)
-    if you do not already have it. For local testing, `dfx >= 0.22.0-beta.0` is
-    required.
--   [x] Clone the example dapp project: `git clone https://github.com/dfinity/examples`
-
 ## Getting started
 
 Sample code for `threshold-schnorr-example` is provided in the [examples repository](https://github.com/dfinity/examples), under either [`/motoko`](https://github.com/dfinity/examples/tree/master/motoko/threshold-schnorr) or [`/rust`](https://github.com/dfinity/examples/tree/master/rust/threshold-schnorr) sub-directories.
 
-### Deploy and test the canister locally 
+### Prerequisites
+This example requires an installation of:
 
-This tutorial will use the Motoko version of the canister.
+- [x] Install the [IC SDK](https://internetcomputer.org/docs/current/developer-docs/setup/install/index.mdx).
+- [x] Clone the example dapp project: `git clone https://github.com/dfinity/examples`
+
+Begin by opening a terminal window.
+
+## Step 1: Setup the project environment
```

Comment:
Now there is just a `Step 1` by itself and there is no `Step 2`. In the threshold-ecdsa/README.md (in this MR here) this was split into two steps.

Reply:
Fixed, thanks